pom.xml

```diff
@@ -163,6 +163,17 @@
         </dependency>
         <!-- == End TERASOLUNA == -->
 
+        <!-- == Begin Password Encode == -->
```

Comment:
- [ ] I think better to change section name `Password Encode` to `Security`. How are you think? @btkobayashirun @bttanakaysd

Reply:
Agree.